### ISSUE TITLE
Custom Domains not ordered to run after Branding

### ISSUE THREAD

**Issue by connor-smith-ag** [auth0/auth0-deploy-cli#789]
### Checklist

- [X] I have looked into the [README](https://github.com/auth0/auth0-deploy-cli#readme) and have not found a suitable solution or answer.
- [X] I have looked into the [documentation](https://github.com/auth0/auth0-deploy-cli#documentation) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/auth0-deploy-cli/issues) and have not found a suitable solution or answer.
- [X] I have upgraded to the [latest version](https://github.com/auth0/auth0-deploy-cli/releases/latest) of this tool and the issue still persists.
- [X] I have searched the [Auth0 Community](https://community.auth0.com/c/sdks/5) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

When importing `tenant.yaml` with both `customDomains` and `branding` present in a fresh tenant, the following error is thrown
```
2023-04-29T08:06:26.358Z - error: Problem running command import during stage processChanges when processing type branding
2023-04-29T08:06:26.359Z - error: This feature requires at least one custom domain to be configured for the tenant.
```

### Expectation

If both `customDomains` and `branding` present, I expect `customDomains` to be processed first, before `branding`.

### Reproduction

`tenant.yaml`
```
customDomains:
  - domain: ##CUSTOM_DOMAIN_ADDRESS##
    primary: true
    status: ready
    tls_policy: recommended
    type: auth0_managed_certs
    verification:
      methods:
        - name: cname
          record: ##CUSTOM_DOMAIN_ADDRESS_CNAME_RECORD##
branding:
  colors:
    page_background: '#000000'
    primary: '#635dff'
  font:
    url: https://use.typekit.net/ugr5xyz.css
  logo_url: ''
  templates:
    - template: universal_login
      body: .\branding_templates\universal_login.html
...
```
error:
```
2023-04-29T08:13:52.691Z - error: Problem running command import during stage processChanges when processing type branding
2023-04-29T08:13:52.691Z - error: This feature requires at least one custom domain to be configured for the tenant.
2023-04-29T08:13:52.692Z - debug: Bad Request: This feature requires at least one custom domain to be configured for the tenant.
```

### Deploy CLI version

7.17.2

### Node version

18.14.2

**Comment by willvedd** [comment]
@connor-smith-ag Thanks for surfacing this request. It is reasonable to expect the custom domains to process before the branding. Controlling the order here is a fairly straightforward code change, but I'll need to look into wether or not the branding settings will immediately recognize the custom domain. 

In the meantime, if this is blocking you, you can leverage the `AUTH0_INCLUDED_ONLY` configuration ([docs](https://github.com/auth0/auth0-deploy-cli/blob/master/docs/configuring-the-deploy-cli.md#auth0_included_only)) to include only custom domains within the tool's purview, import once, then all normal imports thereafter should work.

**Comment by willvedd** [comment]
We've reordered the handlers such that custom domains gets processed before branding settings. Released with [7.17.7](https://github.com/auth0/auth0-deploy-cli/releases/tag/v7.17.7). This _should_ fix OP's issue. However, as noted above, there may be delays in which the custom domain configuration being registered in the backend. I would appreciate you test this out and if problem still persists, we can re-evaluate. However I'm going to close in the meantime. Thanks for opening this ticket!
